Cargo.toml

```diff
@@ -12,7 +12,7 @@ serde = ["dep:serde", "egui/serde"]
 
 [dependencies]
 serde = { version = "1.0", features = ["derive"], optional = true }
-egui = "0.20"
+egui = "0.21"
```

Comment:
Could you disable default features to prevent egui from bundling default fonts?

Reply:
On second thought, disabling default font might be a bad idea because icons are draw using font and they use fancy unicode glyphs. I think  a better solution might be to draw all icons manually but that's a problem for other pr.